Sources/CoreAardvark/Logging/ARKExceptionLogging.m

```diff
@@ -81,7 +81,7 @@ void ARKHandleUncaughtException(NSException *_Nonnull exception)
     }
 }
 
-void ARKEnableLogOnUncaughtException()
```

Comment:
Xcode 14.3 wanted `void` to be added to the function declaration; previous Xcode's were noisy about having `void` added to function prototypes in the .h, so this is, I guess, parity of sorts?